### DISCUSSION TITLE
Permission Required for Admob android

### DISCUSSION THREAD

**Discussion by takhimi** [Poing-Studios/Godot-AdMob-Android-iOS#100] [Q&A]
Hi

What are the permissions required that we need to enabled for Admob on android ?

**Comment by gumaciel** [comment]
Hi, you don't need to enable permissions of Godot to use AdMob, because our Plugin already activates the permissions on code: https://github.com/Poing-Studios/godot-admob-android/blob/master/admob/src/main/AndroidManifest.xml#L4

We only need `android.permission.INTERNET`

**Reply by takhimi** [reply]
Awesome. Thank you for the reply.
